packages/yarnpkg-core/tests/semverUtils.test.ts

```diff
@@ -43,6 +43,9 @@ const SPECS: Specs = [
   [`1.0.0-rc2`, `1.0.0-rc4`, false],
   [`1.0.0-rc4`, `1.0.0-rc2`, false],
   [`1.0.0-beta.1`, `1.0.0-beta.2`, false],
+
+  // These don't match with our patch but do without it
```

Comment:
I don't understand this comment 😅

Reply:
Yeah, the wording is horrible but the gist of it is that before this PR `semverUtils.satisfiesWithPrereleases(version, range)` didn't match while `semver.satisfies(version, range, {includePrerelease: true})` does.